go/api/scripting_and_function_commands_test.go

```diff
@@ -940,3 +940,75 @@ func ExampleGlideClusterClient_ScriptExistsWithRoute() {
 
 	// Output: [true]
 }
+
+func ExampleGlideClient_ScriptKill() {
+	client := getExampleGlideClient()
+
+	// Imagine a script that runs for a long time
+	script := options.NewScript("return 'I am a long-running script'")
+
+	// Execute the script
+	client.InvokeScript(*script)
+
+	// Kill the script while it is running in the 6 seconds
+	_, err := client.ScriptKill()
+	if err != nil {
+		// In this case, our example script should run instantly giving no time
```

Comment:
Maybe omit `InvokeScript` and print `notbusy` error as expected? Like @tjzhang-BQ did for function kill